aiobotocore/endpoint.py

```diff
@@ -37,30 +39,27 @@ async def convert_to_response_dict(http_response, operation_model):
 
     """
     response_dict = {
-        # botocore converts keys to str, so make sure that they are in
-        # the expected case. See detailed discussion here:
-        # https://github.com/aio-libs/aiobotocore/pull/116
-        # aiohttp's CIMultiDict camel cases the headers :(
-        'headers': HTTPHeaderDict(
-            {
-                k.decode('utf-8').lower(): v.decode('utf-8')
-                for k, v in http_response.raw.raw_headers
```

Comment:
it's important we use raw_headers as the non-raw doesn't preserve casing of the value

Reply:
Alright, the alternative is kind of an ugly patch. Let me know what ya think